web/src/components/ToggleButton.tsx

```diff
@@ -76,16 +74,16 @@ export default function ToggleButton({
                 role="button"
                 tabIndex={0}
                 className={twMerge(
-                  'b ml-2 h-6 w-6 select-none justify-center self-center rounded-full bg-white text-center drop-shadow dark:border dark:border-gray-500 dark:bg-gray-900',
+                  'b ml-2 inline-flex h-6 w-6 select-none   justify-center self-center rounded-full bg-white shadow dark:border dark:border-gray-500 dark:bg-gray-900',
```

Comment:
```suggestion
                  'b ml-2 inline-flex h-6 w-6 select-none justify-center self-center rounded-full bg-white shadow dark:border dark:border-gray-500 dark:bg-gray-900',
```
I'm surprised this pass linting, is there a ESLint rule we can enable to remove multiple spaces in classes?

Reply:
There's a few of these around 🙄

Reply:
Unfortunately the Tailwind Prettier plugin removed this check for double spaces as it was causing problems in some other edge cases: https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/71

We might be able to find another eslint plugin just for handling duplicate spaces though :)